authors/apps/articles/views.py

```diff
@@ -63,6 +64,10 @@ def retrieve(self, request, slug):
             article = Article.objects.get(slug=slug)
             serializer = self.serializer_class(
                 article, context={'request': request})
+            if article.author.pk != request.user.pk:
+                username = User.objects.get(username=request.user.username)
```

Comment:
You do not need to get the user model. Request.user provides you with the user model